examples/ASCollectionView/Sample/TextNode.h

```diff
@@ -0,0 +1,15 @@
+//
+//  TextNode.h
+//  Sample
+//
+//  Created by Nikita Ivanchikov on 12/8/15.
+//  Copyright © 2015 Facebook. All rights reserved.
+//
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+@interface TextNode : ASCellNode
```

Comment:
Do we need this as opposed to using ASTextCellNode?  I think it is defined at the bottom of ASCellNode.h.  If that version isn't good enough for testing, e.g. you can't access the internal text node to set the alignment property, let me know and let's improve it in this diff so it is good enough for you to use here / delete this